lib/linux_admin/time_date.rb

```diff
@@ -0,0 +1,21 @@
+module LinuxAdmin
+  class TimeDate
+    extend Common
+    COMMAND = 'timedatectl'
```

Comment:
Would we want to do something like https://github.com/ManageIQ/linux_admin/pull/123#discussion_r40456731 for this too? @Fryguy